firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/internal/InAppMessageStreamManager.java

```diff
@@ -243,7 +243,6 @@ private boolean shouldIgnoreCache(String event) {
                                           "Successfully fetched %d messages from backend",
                                           resp.getMessagesList().size())))
                           .doOnSuccess(analyticsEventsManager::updateContextualTriggers)
-                          .doOnSuccess(abtIntegrationHelper::updateRunningExperiments)
```

Comment:
Why has this been removed?

Reply:
this is the whole point of this PR. This updateRunningExperiments removes the active experiments which are not currently running